ngx-fudis/projects/ngx-fudis/src/lib/components/button/button.component.html

```diff
@@ -1,24 +1,28 @@
 <ng-container *ngIf="tooltip">
   <button
     #buttonElement
+    fudisTooltip
+    [class]="_classList | async"
     [type]="type"
     (blur)="_handleButtonBlur($event)"
     (focus)="_handleButtonFocus($event)"
+    (click)="!disabled && buttonClick($event)"
+    (keydown)="_handleMenuButtonKeyDown($event)"
     [attr.aria-disabled]="disabled"
-    [attr.aria-label]="_ariaLabel"
+    [attr.aria-label]="_ariaLabel | async"
     [attr.id]="_id"
-    (click)="!disabled && buttonClick($event)"
-    [ngClass]="_classList"
-    fudisTooltip
+    [attr.aria-expanded]="!asMenuButton ? null : dropdownOpen"
```

Comment:
Expanded and collapsed state is not transfered properly. Now aria expanded get's state as [object object]. Seems like this state is not properly tied to handleclick function.

Reply:
Currently when focusing on closed menu-button, screen readers reads it as expanded, when should be collapsed.

Reply:
`dropdownOpen` observable was lacking async pipe in the template, fixed.